src/admin/messages/messages.html

```diff
@@ -0,0 +1,73 @@
+<motech-list controller="MessagesController" paginated>
+    <motech-list-header>
+        <div class="btn-group pull-right">
```

Comment:
You should add a button here to show/hide the sidebar. The markup would look similar to
```
<button class="motech-sidebar-button">Filters</button>
```

But it should be a translatable string, and should have a icon of a filter (which you can steal from the email module or the scheduler module)

The css class `.motech-sidebar-button` is looked for by the motech-sidebar directive and add the show/hide functionality.